  INFO root [0x00001654] [2018-06-19 14:21:40,035] - 
  INFO root [0x00001654] [2018-06-19 14:21:40,036] - -----------------------------------------------------------------------------------------


  INFO root [0x00001654] [2018-06-19 14:21:40,038] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001654] [2018-06-19 14:21:40,039] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001654] [2018-06-19 14:21:40,039] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001654] [2018-06-19 14:21:40,053] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001654] [2018-06-19 14:21:40,054] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001654] [2018-06-19 14:21:40,058] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:2638 )
  INFO root [0x00001654] [2018-06-19 14:21:40,058] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:2638, SOMAXCONN=128.
  INFO root [0x00001654] [2018-06-19 14:21:40,059] - server-configs:
  INFO root [0x00001654] [2018-06-19 14:21:40,059] - 	gameUpdateHertz : 10
  INFO root [0x00001654] [2018-06-19 14:21:40,059] - 	entryScriptFile : kbemain
  INFO root [0x00001654] [2018-06-19 14:21:40,059] - 	internalAddr : 192.168.31.56:2638
  INFO root [0x00001654] [2018-06-19 14:21:40,060] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001654] [2018-06-19 14:21:40,060] - 	componentID : 7001
  INFO root [0x00001654] [2018-06-19 14:21:40,755] - Components::findLogger: found logger, addr:192.168.31.56:2627
  INFO root [0x00001654] [2018-06-19 14:21:40,755] - Components::addComponent: reset orderLog, uid=7221!
  INFO root [0x00001654] [2018-06-19 14:21:40,755] - Components::addComponent[logger], uid=7221, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001654] [2018-06-19 14:21:40,762] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 7221. PID: 26920 ----
  INFO root [0x00001654] [2018-06-19 14:21:40,766] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001654] [2018-06-19 14:21:40,766] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001654] [2018-06-19 14:21:40,767] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001654] [2018-06-19 14:21:40,921] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001654] [2018-06-19 14:21:40,922] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001654] [2018-06-19 14:21:40,923] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001654] [2018-06-19 14:21:40,929] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001654] [2018-06-19 14:21:40,930] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001654] [2018-06-19 14:21:40,930] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001654] [2018-06-19 14:21:40,931] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001654] [2018-06-19 14:21:40,991] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001654] [2018-06-19 14:21:41,044] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001654] [2018-06-19 14:21:41,065] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001654] [2018-06-19 14:21:41,075] - loaded script:Account(1).
  INFO root [0x00001654] [2018-06-19 14:21:41,075] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001654] [2018-06-19 14:21:41,077] - TelnetServer server is running on port 40000
  INFO root [0x00001654] [2018-06-19 14:21:41,668] - ---- baseapp is running ----
 DEBUG root [0x00001654] [2018-06-19 14:21:41,668] - Components::process(): Request for the process of identity...
  WARN root [0x00001654] [2018-06-19 14:21:47,954] - Forwarding logs to logger[192.168.31.56:2627/0/0/0]...
 ERROR root [0x00001654] [2018-06-19 16:43:18,441] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001654] [2018-06-19 16:43:18,489] - The following logs sent to logger failed:
  WARN root [0x00001654] [2018-06-19 16:43:18,576] - ==> [2018-06-19 16:43:18 397] TCPPacketReceiver::processPendingEvents(192.168.31.56:2627): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001654] [2018-06-19 16:43:18,576] - ==> [2018-06-19 16:43:18 442] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001654] [2018-06-19 16:43:18,576] - TCPPacketReceiver::processPendingEvents(192.168.31.56:2631): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001654] [2018-06-19 16:43:18,577] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001654] [2018-06-19 16:43:18,622] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001654] [2018-06-19 16:43:18,623] - onBaseAppShutDown: state=0
  INFO root [0x00001654] [2018-06-19 16:43:18,623] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001654] [2018-06-19 16:43:18,623] - onBaseAppShutDown: state=1
  INFO root [0x00001654] [2018-06-19 16:43:18,624] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001654] [2018-06-19 16:43:18,624] - onBaseAppShutDown: state=2
 DEBUG root [0x00001654] [2018-06-19 16:43:18,804] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001654] [2018-06-19 16:43:18,804] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001654] [2018-06-19 16:43:18,804] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001654] [2018-06-19 16:43:18,805] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001654] [2018-06-19 16:43:18,805] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001654] [2018-06-19 16:43:18,806] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001654] [2018-06-19 16:43:18,806] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001654] [2018-06-19 16:43:18,806] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001654] [2018-06-19 16:43:18,854] - Script::uninstall(): is successfully!
 DEBUG root [0x00001654] [2018-06-19 16:43:18,864] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00006590] [2018-06-19 16:43:27,143] - 
  INFO root [0x00006590] [2018-06-19 16:43:27,144] - -----------------------------------------------------------------------------------------


  INFO root [0x00006590] [2018-06-19 16:43:27,145] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00006590] [2018-06-19 16:43:27,146] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00006590] [2018-06-19 16:43:27,147] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00006590] [2018-06-19 16:43:27,157] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00006590] [2018-06-19 16:43:27,158] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00006590] [2018-06-19 16:43:27,162] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:5966 )
  INFO root [0x00006590] [2018-06-19 16:43:27,163] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:5966, SOMAXCONN=128.
  INFO root [0x00006590] [2018-06-19 16:43:27,163] - server-configs:
  INFO root [0x00006590] [2018-06-19 16:43:27,163] - 	gameUpdateHertz : 10
  INFO root [0x00006590] [2018-06-19 16:43:27,163] - 	entryScriptFile : kbemain
  INFO root [0x00006590] [2018-06-19 16:43:27,164] - 	internalAddr : 192.168.31.56:5966
  INFO root [0x00006590] [2018-06-19 16:43:27,164] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00006590] [2018-06-19 16:43:27,164] - 	componentID : 7001
  INFO root [0x00006590] [2018-06-19 16:43:29,326] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 2247. PID: 27076 ----
  INFO root [0x00006590] [2018-06-19 16:43:29,360] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00006590] [2018-06-19 16:43:29,360] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00006590] [2018-06-19 16:43:29,362] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00006590] [2018-06-19 16:43:29,611] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00006590] [2018-06-19 16:43:29,612] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00006590] [2018-06-19 16:43:29,613] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00006590] [2018-06-19 16:43:29,621] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00006590] [2018-06-19 16:43:29,622] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00006590] [2018-06-19 16:43:29,622] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00006590] [2018-06-19 16:43:29,624] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00006590] [2018-06-19 16:43:29,829] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00006590] [2018-06-19 16:43:30,044] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00006590] [2018-06-19 16:43:30,045] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00006590] [2018-06-19 16:43:30,210] - loaded script:Account(1).
  INFO root [0x00006590] [2018-06-19 16:43:30,211] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00006590] [2018-06-19 16:43:30,212] - TelnetServer server is running on port 40000
  INFO root [0x00006590] [2018-06-19 16:43:31,199] - ---- baseapp is running ----
 DEBUG root [0x00006590] [2018-06-19 16:43:31,200] - Components::process(): Request for the process of identity...
  WARN root [0x00006590] [2018-06-19 16:43:37,316] - EntityApp::checkTickPeriod: tick took 6115% (6.11 seconds)!
  INFO root [0x00006590] [2018-06-19 16:43:38,348] - Components::findComponents: find logger(1)...
  INFO root [0x00006590] [2018-06-19 16:43:38,372] - Components::findComponents: found logger, addr:192.168.31.56:5968
  INFO root [0x00006590] [2018-06-19 16:43:38,373] - Components::addComponent: reset orderLog, uid=2247!
  INFO root [0x00006590] [2018-06-19 16:43:38,373] - Components::addComponent[logger], uid=2247, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00006590] [2018-06-19 16:43:38,375] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00006590] [2018-06-19 16:43:38,479] - Components::findComponents: found dbmgr, addr:192.168.31.56:5973
  INFO root [0x00006590] [2018-06-19 16:43:38,479] - Components::addComponent: reset orderLog, uid=2247!
  INFO root [0x00006590] [2018-06-19 16:43:38,480] - Components::addComponent[dbmgr], uid=2247, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00006590] [2018-06-19 16:43:38,480] - Components::findComponents: found dbmgr, addr:192.168.31.56:5973
  WARN root [0x00006590] [2018-06-19 16:43:39,480] - Forwarding logs to logger[192.168.31.56:5968/0/0/0]...
 ERROR root [0x00006590] [2018-06-19 17:57:17,176] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00006590] [2018-06-19 17:57:17,189] - The following logs sent to logger failed:
  WARN root [0x00006590] [2018-06-19 17:57:17,190] - ==> [2018-06-19 17:57:17 172] TCPPacketReceiver::processPendingEvents(192.168.31.56:5968): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00006590] [2018-06-19 17:57:17,190] - ==> [2018-06-19 17:57:17 177] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00006590] [2018-06-19 17:57:17,300] - TCPPacketReceiver::processPendingEvents(192.168.31.56:5973): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00006590] [2018-06-19 17:57:17,303] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00006590] [2018-06-19 17:57:17,309] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00006590] [2018-06-19 17:57:17,313] - onBaseAppShutDown: state=0
  INFO root [0x00006590] [2018-06-19 17:57:17,313] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00006590] [2018-06-19 17:57:17,314] - onBaseAppShutDown: state=1
  INFO root [0x00006590] [2018-06-19 17:57:17,314] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00006590] [2018-06-19 17:57:17,314] - onBaseAppShutDown: state=2
 DEBUG root [0x00006590] [2018-06-19 17:57:17,354] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00006590] [2018-06-19 17:57:17,354] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00006590] [2018-06-19 17:57:17,355] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00006590] [2018-06-19 17:57:17,355] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00006590] [2018-06-19 17:57:17,355] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00006590] [2018-06-19 17:57:17,356] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00006590] [2018-06-19 17:57:17,356] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00006590] [2018-06-19 17:57:17,356] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00006590] [2018-06-19 17:57:17,378] - Script::uninstall(): is successfully!
 DEBUG root [0x00006590] [2018-06-19 17:57:17,379] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002abc] [2018-06-19 17:57:23,874] - 
  INFO root [0x00002abc] [2018-06-19 17:57:23,884] - -----------------------------------------------------------------------------------------


  INFO root [0x00002abc] [2018-06-19 17:57:23,890] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002abc] [2018-06-19 17:57:23,891] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002abc] [2018-06-19 17:57:23,892] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002abc] [2018-06-19 17:57:23,937] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00002abc] [2018-06-19 17:57:23,937] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00002abc] [2018-06-19 17:57:23,942] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:6995 )
  INFO root [0x00002abc] [2018-06-19 17:57:23,942] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:6995, SOMAXCONN=128.
  INFO root [0x00002abc] [2018-06-19 17:57:23,943] - server-configs:
  INFO root [0x00002abc] [2018-06-19 17:57:23,943] - 	gameUpdateHertz : 10
  INFO root [0x00002abc] [2018-06-19 17:57:23,943] - 	entryScriptFile : kbemain
  INFO root [0x00002abc] [2018-06-19 17:57:23,943] - 	internalAddr : 192.168.31.56:6995
  INFO root [0x00002abc] [2018-06-19 17:57:23,944] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00002abc] [2018-06-19 17:57:23,944] - 	componentID : 7001
  INFO root [0x00002abc] [2018-06-19 17:57:25,974] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 16733. PID: 1692 ----
  INFO root [0x00002abc] [2018-06-19 17:57:25,975] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002abc] [2018-06-19 17:57:25,976] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002abc] [2018-06-19 17:57:25,977] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,195] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,197] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,198] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,205] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,206] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,206] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,208] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,397] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002abc] [2018-06-19 17:57:26,591] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002abc] [2018-06-19 17:57:26,592] - Script::install(): KBExtra is successfully!
 ERROR root [0x00002abc] [2018-06-19 17:57:26,676] - EntityDef::initialize: Could not load module[Account]
 S_ERR root [0x00002abc] [2018-06-19 17:57:26,746] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/base\Account.py", line 43
    if name=='':
               ^
SyntaxError: invalid syntax
 ERROR root [0x00002abc] [2018-06-19 17:57:26,747] - app::initialize(): initialization failed!
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,749] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,749] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,750] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,750] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,751] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,751] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,751] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,752] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002abc] [2018-06-19 17:57:26,772] - Script::uninstall(): is successfully!
 DEBUG root [0x00002abc] [2018-06-19 17:57:26,772] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002abc] [2018-06-19 17:57:27,095] - ThreadPool::destroy(): waiting for thread(2)[0x4342c0,0x434258,], try=1
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,518] - TPThread::~TPThread(): 0x4342c0
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,519] - TPThread::~TPThread(): 0x434258
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,519] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,520] - Bundle::destroyObjPool(): size 3.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,520] - Channel::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,521] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,521] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,521] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,522] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,522] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-06-19 17:57:27,522] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [0x00002abc] [2018-06-19 17:57:34,051] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x00003964] [2018-06-19 17:57:45,562] - 
  INFO root [0x00003964] [2018-06-19 17:57:45,563] - -----------------------------------------------------------------------------------------


  INFO root [0x00003964] [2018-06-19 17:57:45,565] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00003964] [2018-06-19 17:57:45,566] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00003964] [2018-06-19 17:57:45,567] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00003964] [2018-06-19 17:57:45,587] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00003964] [2018-06-19 17:57:45,588] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00003964] [2018-06-19 17:57:45,593] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7025 )
  INFO root [0x00003964] [2018-06-19 17:57:45,593] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7025, SOMAXCONN=128.
  INFO root [0x00003964] [2018-06-19 17:57:45,593] - server-configs:
  INFO root [0x00003964] [2018-06-19 17:57:45,594] - 	gameUpdateHertz : 10
  INFO root [0x00003964] [2018-06-19 17:57:45,594] - 	entryScriptFile : kbemain
  INFO root [0x00003964] [2018-06-19 17:57:45,594] - 	internalAddr : 192.168.31.56:7025
  INFO root [0x00003964] [2018-06-19 17:57:45,595] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00003964] [2018-06-19 17:57:45,595] - 	componentID : 7001
  INFO root [0x00003964] [2018-06-19 17:57:46,550] - Components::findLogger: found logger, addr:192.168.31.56:7020
  INFO root [0x00003964] [2018-06-19 17:57:46,551] - Components::addComponent: reset orderLog, uid=16805!
  INFO root [0x00003964] [2018-06-19 17:57:46,551] - Components::addComponent[logger], uid=16805, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003964] [2018-06-19 17:57:46,561] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 16805. PID: 29160 ----
  INFO root [0x00003964] [2018-06-19 17:57:46,562] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00003964] [2018-06-19 17:57:46,564] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003964] [2018-06-19 17:57:46,565] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00003964] [2018-06-19 17:57:46,718] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003964] [2018-06-19 17:57:46,719] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00003964] [2018-06-19 17:57:46,720] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00003964] [2018-06-19 17:57:46,727] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003964] [2018-06-19 17:57:46,727] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00003964] [2018-06-19 17:57:46,728] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003964] [2018-06-19 17:57:46,729] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003964] [2018-06-19 17:57:46,805] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00003964] [2018-06-19 17:57:46,871] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00003964] [2018-06-19 17:57:46,871] - Script::install(): KBExtra is successfully!
 ERROR root [0x00003964] [2018-06-19 17:57:46,893] - EntityDef::initialize: Could not load module[Account]
 S_ERR root [0x00003964] [2018-06-19 17:57:46,895] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/base\Account.py", line 43
    if name=='':
               ^
SyntaxError: invalid syntax
 ERROR root [0x00003964] [2018-06-19 17:57:46,897] - app::initialize(): initialization failed!
 DEBUG root [0x00003964] [2018-06-19 17:57:46,899] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003964] [2018-06-19 17:57:46,899] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00003964] [2018-06-19 17:57:46,899] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00003964] [2018-06-19 17:57:46,900] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003964] [2018-06-19 17:57:46,900] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00003964] [2018-06-19 17:57:46,900] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003964] [2018-06-19 17:57:46,900] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003964] [2018-06-19 17:57:46,900] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00003964] [2018-06-19 17:57:46,913] - Script::uninstall(): is successfully!
 DEBUG root [0x00003964] [2018-06-19 17:57:46,913] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00003964] [2018-06-19 17:57:47,213] - ThreadPool::destroy(): waiting for thread(2)[0x19fcdf8,0x19fcd90,], try=1
 DEBUG root [0x00003964] [2018-06-19 17:57:47,615] - TPThread::~TPThread(): 0x19fcdf8
 DEBUG root [0x00003964] [2018-06-19 17:57:47,615] - TPThread::~TPThread(): 0x19fcd90
 DEBUG root [0x00003964] [2018-06-19 17:57:47,615] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00003964] [2018-06-19 17:57:47,616] - Bundle::destroyObjPool(): size 0.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,617] - Channel::destroyObjPool(): size 15.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,618] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,619] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,619] - EndPoint::destroyObjPool(): size 15.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,619] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,620] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00003964] [2018-06-19 17:57:47,620] - UDPPacketReceiver::destroyObjPool(): size 0.
  WARN root [0x00003964] [2018-06-19 17:57:47,621] - Forwarding logs to logger[192.168.31.56:7020/0/0/0]...
  INFO root [0x00003964] [2018-06-19 17:57:54,137] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x00006cb0] [2018-06-19 18:01:19,367] - 
  INFO root [0x00006cb0] [2018-06-19 18:01:19,369] - -----------------------------------------------------------------------------------------


  INFO root [0x00006cb0] [2018-06-19 18:01:19,371] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00006cb0] [2018-06-19 18:01:19,372] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00006cb0] [2018-06-19 18:01:19,373] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00006cb0] [2018-06-19 18:01:19,398] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00006cb0] [2018-06-19 18:01:19,398] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00006cb0] [2018-06-19 18:01:19,405] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7089 )
  INFO root [0x00006cb0] [2018-06-19 18:01:19,405] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7089, SOMAXCONN=128.
  INFO root [0x00006cb0] [2018-06-19 18:01:19,406] - server-configs:
  INFO root [0x00006cb0] [2018-06-19 18:01:19,406] - 	gameUpdateHertz : 10
  INFO root [0x00006cb0] [2018-06-19 18:01:19,406] - 	entryScriptFile : kbemain
  INFO root [0x00006cb0] [2018-06-19 18:01:19,407] - 	internalAddr : 192.168.31.56:7089
  INFO root [0x00006cb0] [2018-06-19 18:01:19,407] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00006cb0] [2018-06-19 18:01:19,408] - 	componentID : 7001
  INFO root [0x00006cb0] [2018-06-19 18:01:20,324] - Components::findLogger: found logger, addr:192.168.31.56:7083
  INFO root [0x00006cb0] [2018-06-19 18:01:20,325] - Components::addComponent: reset orderLog, uid=17504!
  INFO root [0x00006cb0] [2018-06-19 18:01:20,326] - Components::addComponent[logger], uid=17504, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00006cb0] [2018-06-19 18:01:20,335] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 17504. PID: 10988 ----
  INFO root [0x00006cb0] [2018-06-19 18:01:20,335] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00006cb0] [2018-06-19 18:01:20,336] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00006cb0] [2018-06-19 18:01:20,337] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,490] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,490] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,491] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,497] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,497] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,498] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,499] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,559] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00006cb0] [2018-06-19 18:01:20,610] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00006cb0] [2018-06-19 18:01:20,610] - Script::install(): KBExtra is successfully!
 ERROR root [0x00006cb0] [2018-06-19 18:01:20,625] - EntityDef::initialize: Could not load module[Account]
 S_ERR root [0x00006cb0] [2018-06-19 18:01:20,626] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/base\Account.py", line 43
    if name=="":
               ^
SyntaxError: invalid syntax
 ERROR root [0x00006cb0] [2018-06-19 18:01:20,627] - app::initialize(): initialization failed!
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,628] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,629] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,629] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,629] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,629] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,629] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,629] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,630] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00006cb0] [2018-06-19 18:01:20,642] - Script::uninstall(): is successfully!
 DEBUG root [0x00006cb0] [2018-06-19 18:01:20,642] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00006cb0] [2018-06-19 18:01:20,942] - ThreadPool::destroy(): waiting for thread(2)[0x8442c0,0x844258,], try=1
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,402] - TPThread::~TPThread(): 0x8442c0
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,403] - TPThread::~TPThread(): 0x844258
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,403] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,404] - Bundle::destroyObjPool(): size 0.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,404] - Channel::destroyObjPool(): size 15.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,405] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,406] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,406] - EndPoint::destroyObjPool(): size 15.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,406] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,407] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00006cb0] [2018-06-19 18:01:21,407] - UDPPacketReceiver::destroyObjPool(): size 0.
  WARN root [0x00006cb0] [2018-06-19 18:01:21,408] - Forwarding logs to logger[192.168.31.56:7083/0/0/0]...
  INFO root [0x00006cb0] [2018-06-19 18:01:27,947] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x00007064] [2018-06-19 18:01:41,988] - 
  INFO root [0x00007064] [2018-06-19 18:01:41,993] - -----------------------------------------------------------------------------------------


  INFO root [0x00007064] [2018-06-19 18:01:41,995] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00007064] [2018-06-19 18:01:41,996] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00007064] [2018-06-19 18:01:41,997] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00007064] [2018-06-19 18:01:42,025] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00007064] [2018-06-19 18:01:42,026] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00007064] [2018-06-19 18:01:42,033] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7112 )
  INFO root [0x00007064] [2018-06-19 18:01:42,034] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7112, SOMAXCONN=128.
  INFO root [0x00007064] [2018-06-19 18:01:42,034] - server-configs:
  INFO root [0x00007064] [2018-06-19 18:01:42,035] - 	gameUpdateHertz : 10
  INFO root [0x00007064] [2018-06-19 18:01:42,035] - 	entryScriptFile : kbemain
  INFO root [0x00007064] [2018-06-19 18:01:42,035] - 	internalAddr : 192.168.31.56:7112
  INFO root [0x00007064] [2018-06-19 18:01:42,036] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00007064] [2018-06-19 18:01:42,036] - 	componentID : 7001
  INFO root [0x00007064] [2018-06-19 18:01:42,934] - Components::findLogger: found logger, addr:192.168.31.56:7108
  INFO root [0x00007064] [2018-06-19 18:01:42,934] - Components::addComponent: reset orderLog, uid=17575!
  INFO root [0x00007064] [2018-06-19 18:01:42,935] - Components::addComponent[logger], uid=17575, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00007064] [2018-06-19 18:01:42,950] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 17575. PID: 27056 ----
  INFO root [0x00007064] [2018-06-19 18:01:42,952] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00007064] [2018-06-19 18:01:42,958] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00007064] [2018-06-19 18:01:42,958] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00007064] [2018-06-19 18:01:43,123] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00007064] [2018-06-19 18:01:43,124] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00007064] [2018-06-19 18:01:43,125] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00007064] [2018-06-19 18:01:43,132] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00007064] [2018-06-19 18:01:43,133] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00007064] [2018-06-19 18:01:43,133] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00007064] [2018-06-19 18:01:43,135] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00007064] [2018-06-19 18:01:43,211] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00007064] [2018-06-19 18:01:43,275] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00007064] [2018-06-19 18:01:43,275] - Script::install(): KBExtra is successfully!
 ERROR root [0x00007064] [2018-06-19 18:01:43,295] - EntityDef::initialize: Could not load module[Account]
 S_ERR root [0x00007064] [2018-06-19 18:01:43,297] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/base\Account.py", line 43
    if name=="":
               ^
SyntaxError: invalid syntax
 ERROR root [0x00007064] [2018-06-19 18:01:43,299] - app::initialize(): initialization failed!
 DEBUG root [0x00007064] [2018-06-19 18:01:43,301] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00007064] [2018-06-19 18:01:43,301] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00007064] [2018-06-19 18:01:43,302] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00007064] [2018-06-19 18:01:43,302] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00007064] [2018-06-19 18:01:43,302] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00007064] [2018-06-19 18:01:43,303] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00007064] [2018-06-19 18:01:43,303] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00007064] [2018-06-19 18:01:43,303] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00007064] [2018-06-19 18:01:43,322] - Script::uninstall(): is successfully!
 DEBUG root [0x00007064] [2018-06-19 18:01:43,323] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00007064] [2018-06-19 18:01:43,649] - ThreadPool::destroy(): waiting for thread(2)[0x66cdf8,0x66cd90,], try=1
 DEBUG root [0x00007064] [2018-06-19 18:01:44,082] - TPThread::~TPThread(): 0x66cdf8
 DEBUG root [0x00007064] [2018-06-19 18:01:44,083] - TPThread::~TPThread(): 0x66cd90
 DEBUG root [0x00007064] [2018-06-19 18:01:44,083] - ThreadPool::destroy(): successfully!
 DEBUG root [0x00007064] [2018-06-19 18:01:44,084] - Bundle::destroyObjPool(): size 0.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,085] - Channel::destroyObjPool(): size 15.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,086] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,086] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,087] - EndPoint::destroyObjPool(): size 15.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,087] - Address::destroyObjPool(): size 0.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,087] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [0x00007064] [2018-06-19 18:01:44,088] - UDPPacketReceiver::destroyObjPool(): size 0.
  WARN root [0x00007064] [2018-06-19 18:01:44,088] - Forwarding logs to logger[192.168.31.56:7108/0/0/0]...
  INFO root [0x00007064] [2018-06-19 18:01:50,632] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [0x000069c8] [2018-06-19 18:02:41,065] - 
  INFO root [0x000069c8] [2018-06-19 18:02:41,066] - -----------------------------------------------------------------------------------------


  INFO root [0x000069c8] [2018-06-19 18:02:41,068] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000069c8] [2018-06-19 18:02:41,069] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000069c8] [2018-06-19 18:02:41,069] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000069c8] [2018-06-19 18:02:41,082] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x000069c8] [2018-06-19 18:02:41,083] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x000069c8] [2018-06-19 18:02:41,087] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7152 )
  INFO root [0x000069c8] [2018-06-19 18:02:41,087] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7152, SOMAXCONN=128.
  INFO root [0x000069c8] [2018-06-19 18:02:41,088] - server-configs:
  INFO root [0x000069c8] [2018-06-19 18:02:41,088] - 	gameUpdateHertz : 10
  INFO root [0x000069c8] [2018-06-19 18:02:41,088] - 	entryScriptFile : kbemain
  INFO root [0x000069c8] [2018-06-19 18:02:41,089] - 	internalAddr : 192.168.31.56:7152
  INFO root [0x000069c8] [2018-06-19 18:02:41,089] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x000069c8] [2018-06-19 18:02:41,089] - 	componentID : 7001
  INFO root [0x000069c8] [2018-06-19 18:02:42,111] - Components::findLogger: found logger, addr:192.168.31.56:7151
  INFO root [0x000069c8] [2018-06-19 18:02:42,112] - Components::addComponent: reset orderLog, uid=17768!
  INFO root [0x000069c8] [2018-06-19 18:02:42,113] - Components::addComponent[logger], uid=17768, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000069c8] [2018-06-19 18:02:42,122] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 17768. PID: 30504 ----
  INFO root [0x000069c8] [2018-06-19 18:02:42,122] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000069c8] [2018-06-19 18:02:42,123] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000069c8] [2018-06-19 18:02:42,123] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,269] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,269] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,270] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,275] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,276] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,276] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,277] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,335] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000069c8] [2018-06-19 18:02:42,389] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000069c8] [2018-06-19 18:02:42,389] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000069c8] [2018-06-19 18:02:42,408] - loaded script:Account(1).
  INFO root [0x000069c8] [2018-06-19 18:02:42,408] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000069c8] [2018-06-19 18:02:42,410] - TelnetServer server is running on port 40000
  INFO root [0x000069c8] [2018-06-19 18:02:43,545] - ---- baseapp is running ----
 DEBUG root [0x000069c8] [2018-06-19 18:02:43,546] - Components::process(): Request for the process of identity...
  WARN root [0x000069c8] [2018-06-19 18:02:49,952] - Forwarding logs to logger[192.168.31.56:7151/0/0/0]...
 ERROR root [0x000069c8] [2018-06-19 23:29:07,566] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000069c8] [2018-06-19 23:29:07,614] - The following logs sent to logger failed:
  WARN root [0x000069c8] [2018-06-19 23:29:07,689] - ==> [2018-06-19 23:29:07 516] TCPPacketReceiver::processPendingEvents(192.168.31.56:7151): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000069c8] [2018-06-19 23:29:07,690] - ==> [2018-06-19 23:29:07 567] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000069c8] [2018-06-19 23:29:07,690] - EntityApp::checkTickPeriod: tick took 258% (0.26 seconds)!
  WARN root [0x000069c8] [2018-06-19 23:29:07,691] - TCPPacketReceiver::processPendingEvents(192.168.31.56:7146): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000069c8] [2018-06-19 23:29:07,691] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000069c8] [2018-06-19 23:29:07,747] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000069c8] [2018-06-19 23:29:07,747] - onBaseAppShutDown: state=0
  INFO root [0x000069c8] [2018-06-19 23:29:07,748] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000069c8] [2018-06-19 23:29:07,748] - onBaseAppShutDown: state=1
  INFO root [0x000069c8] [2018-06-19 23:29:07,748] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000069c8] [2018-06-19 23:29:07,748] - onBaseAppShutDown: state=2
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,920] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,920] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,920] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,920] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,921] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,921] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,921] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000069c8] [2018-06-19 23:29:07,921] - Script::uninstall_py_dlls(): PyInit_unicodedata
